Assets/PatchKit Patcher/Scripts/AppData/Local/ZipUnarchiver.cs

```diff
@@ -43,19 +43,19 @@ public void Unarchive(CancellationToken cancellationToken)
             {
                 zip.Password = _password;
 
-                int entry = 0;
-
-                OnUnarchiveProgressChanged(null, false, 0, zip.Count, 0.0);
+                int entry = 1;
 
                 foreach (var zipEntry in zip)
                 {
+                    OnUnarchiveProgressChanged(zipEntry.FileName, !zipEntry.FileName.EndsWith("/"), entry, zip.Count, 0.0);
```

Comment:
Maybe it may be a good idea to move `zipEntry.FileName.EndsWith("/")` to external util class? Like `Paths.IsDirectory(zipEntry.FileName)`.

Reply:
Actually, could it simply be replaced by `zipEntry.IsDirectory` ?

Reply:
maybe?

Reply:
I've actually discovered that zip library has built-in way for checking whether entry is directory or file.